wpeverest/everest-forms
# Email Notifications and Smart Tags enhancement

### All Submissions:






### Changes proposed in this Pull Request:



Closes #81 #112 #114  .

### How to test the changes in this Pull Request:

1. Select a form and go to the settings tab
2. Go on Email setting and check the smart tags you will see the fixes of the reported issue as I have mentioned above

### Types of changes:




### Other information:




### Changelog entry

> Fix - Fill email notifications settings while blank form.
> Fix - Smart tag design issue
> Tweak - Smart Tags enhancement.
